frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -103,7 +101,7 @@ export function AdminScreen({
                     Select a precinct for this device…
                   </option>
                   <option value={ALL_PRECINCTS_OPTION_VALUE}>
-                    {AllPrecinctsDisplayName}
+                    All Precincts
```

Comment:
Didn't you make a constant for this in the precinct selection utils?

Reply:
Yes, I just didn't bother importing it here. Can do so.

Reply:
Switched.